x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/field_select.tsx

```diff
@@ -168,60 +171,73 @@ export function FieldSelect({
     existingFields,
     markAllFieldsCompatible,
   ]);
+  const comboBoxRef = useRef<HTMLInputElement>(null);
+  const labelWidth =
+    (comboBoxRef.current?.clientWidth || DEFAULT_COMBOBOX_WIDTH) - COMBOBOX_PADDINGS;
+  const font = comboBoxRef.current
+    ? window.getComputedStyle(comboBoxRef.current).font
+    : DEFAULT_FONT;
```

Comment:
Seems like this should be read once, not on every render. I don't think changing the window size would cause a new font?